### ISSUE TITLE
Cannot read property 'focus' of null

### ISSUE THREAD

**Issue by jrburke** [ember-a11y/ember-a11y#68]
Intermittent: After some time in the background, coming back to the the web app in a Chrome tab, clicking on something to navigate to a new route, focusing.js throws an error:

```
TypeError: Cannot read property 'focus' of null
```

### Reproduction

Ember 2.18.2
ember-a11y: 0.1.15

It is hard for us to reproduce. Maybe happened 4-5 times in the last month on a developer's machine. It seems to occur during longer sessions with the app:

1. launch the app in Chrome
2. change focus to another program (i.e. Atom)
3. return to Chrome ~5-10 mins later and start navigating around the app. 

The crash occurs immediately after navigating to a new page (i.e. when focusing outlet attempts to shift focus).

### Actual

Error is thrown:

![focusing-error](https://user-images.githubusercontent.com/73359/44805682-c0de8080-ab79-11e8-90a2-5ba5edc53f6d.png)

### Expected

No error is thrown.

Looking at the section of code in focusing.js:

```javascript
      Ember.run.schedule('afterRender', this, function() {
        this.element.blur();
        Ember.run.next(this, function() {
          this.element.focus();
        });
});
```

I wonder if the the next() call is run after a point where that particular focusing-element has been destroyed? Should there be an `if (this.element)` check before the `.focus()` call? I wonder if Chrome is slowing down timers while the tab is in the background and that creates more of an opportunity for the failure to manifest.

**Comment by nathanhammond** [comment]
Regardless of the root cause, this should likely be behind a guard. I'll make that change.

**Comment by nathanhammond** [comment]
Fixed by #70, will be in `ember-a11y@0.1.17`, released tonight.

**Comment by jrburke** [comment]
Awesome! Thank you very much!
